docs/_guides/codewind-vs-code-quick-guide.md

```diff
@@ -82,10 +82,11 @@ Your application is now opened in a browser, showing the welcome to your Appsody
 
 ### Adding a REST service to your application
  1. Go to your project's workspace under the Explorer tab.
- 2. Navigate to `src->main->java->dev->appsody->starter`.
+ 2. Go to `src->main->java->dev->appsody->starter`.
```

Comment:
Format of arrows should match style guide. https://github.com/eclipse/codewind-docs/wiki/Writing-style-guidelines#arrows
Code phrase format might be appropriate here.